src/Symfony/Component/Notifier/Bridge/LineNotify/LineNotifyTransport.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineNotify;
+
+use Symfony\Component\Notifier\Exception\LengthException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Akira Kurozumi <info@a-zumi.net>
+ */
+final class LineNotifyTransport extends AbstractTransport
+{
+    protected const HOST = 'notify-api.line.me';
+
+    private const SUBJECT_LIMIT = 1000;
```

Comment:
Doesn't the API throw a proper error when exceeding the limit? If yes I am against a username validation here, because it can change in the future on LINE side